src/main/resources/db/changesets/20210301_revised_initial_schema.yaml

```diff
@@ -0,0 +1,246 @@
+# "Rebasing" the schema to a single changeset prior to production
+databaseChangeLog:
+- changeSet:
+    id: schema revision 2021
+    author: dd
+    changes:
+    - createTable:
+        tableName: workspace
+        remarks: |
+          There is one row in the workspace table for each workspace.
+        columns:
+          - column:
+              name: workspace_id
+              type: text
```

Comment:
Is there not a UUID type?

Reply:
I looked and there is a UUID type. I wonder if there are any downsides to using it.

Reply:
I looked at both the uuid and the enum types. When you use them, you are at the mercy of middle layers of JDBC and the postgres driver to get the transform right. I think we are better off keeping it vanilla.